### DISCUSSION TITLE
Articles

### DISCUSSION THREAD

**Discussion by muthuannamalai12** [EddieHubCommunity/support#1797] [Show and tell]
My new blog is out. This article talks about What is HTML and the basics of it. Consider giving it a clap 👏 if you like it:

https://muthuannamalai.hashnode.dev/basics-of-html-and-everything-you-need-to-know-about-it

**Comment by muthuannamalai12** [comment]
My new blog is out. This article talks about Top 10 Technology Trends of 2021.Consider giving it a clap 👏 if you like it:

https://muthuannamalai.hashnode.dev/top-10-technology-trends-of-2021

**Comment by muthuannamalai12** [comment]
My new blog is out. This article talks about Building your Portfolio only using HTML. Consider giving it a clap 👏 if you like it:

https://muthuannamalai.hashnode.dev/web-development-build-your-portfolio-website-design-using-html

**Comment by muthuannamalai12** [comment]
My new blog is out. This article talks about 5 crazy future tech trends that will define the future. Consider giving it a clap 👏 if you like it:

https://muthuannamalai.hashnode.dev/5-crazy-tech-trends-that-will-define-the-future

**Comment by muthuannamalai12** [comment]
My new blog is out. This article talks about how to contribute to open-source as a beginner . Consider giving it a clap 👏 if you like it:

https://muthuannamalai.hashnode.dev/how-to-contribute-to-open-source-projects-as-a-beginner

**Comment by niteshseram** [comment]
I wrote my first blog. Give it a read. I am not professional just sharing what I am learning. Do correct me if I am somewhere wrong. If you like it, give it a reaction
https://blog.niteshseram.in/what-is-docker

**Reply by kaiwalyakoparkar** [reply]
Hey! Great blog but I would like to give some suggestions. These are my two cents:
- Add a greeting to your blog instead of directly jumping onto the topic
- Provide a suitable outtro
- Provide resources to follow (Something you might want to refer your reader)

**Comment by muthuannamalai12** [comment]
My new blog is out. This article talks about Can Tesla Succeed in India. Consider giving it a clap if you like it:

https://muthuannamalai.hashnode.dev/could-tesla-succeed-in-india

**Comment by muthuannamalai12** [comment]
My new blog is out. This article talks about 10+ Open Source programs students can apply to .Consider giving it a clap 👏 if you like it:

https://muthuannamalai.hashnode.dev/10-open-source-programs-students-can-apply-to

**Comment by muthuannamalai12** [comment]
My new blog is out. This article talks about everything you need to know about Hackathons. Consider giving it a clap if you like it:

https://muthuannamalai.hashnode.dev/everything-you-need-to-know-about-hackathons

**Comment by muthuannamalai12** [comment]
My new blog is out. This article talks about the various themes of VS Code. Consider giving it a clap 👏 if you like it:

https://muthuannamalai.hashnode.dev/top-10-vs-code-themes

**Comment by Prathamesh-Shanbhag** [comment]
Great articles Muthu Keep going! 👏👏👏

**Reply by muthuannamalai12** [reply]
Thank you Prathamesh

**Comment by muthuannamalai12** [comment]
My new blog is out. Creating an impressive portfolio is not easy. If you need a spark to ignite your creativity then take a look at my article. Consider giving it a clap if you like it:

https://muthuannamalai.hashnode.dev/15-developer-portfolios-to-inspire-you
